src/traces/sunburst/constants.js

```diff
@@ -12,12 +12,12 @@ module.exports = {
     CLICK_TRANSITION_TIME: 750,
     CLICK_TRANSITION_EASING: 'linear',
     eventDataKeys: [
-        'parentLabel',
-        'visibleLabel',
-        'rootLabel',
-        'percentParent',
-        'percentVisible',
+        'currentPath',
         'percentRoot',
-        'currentPath'
+        'percentVisible',
+        'percentParent',
+        'root',
+        'visible',
```

Comment:
We'll need to find a better. `visible` already means trace visibility.

Reply:
100% agree there.
Also using `parent` here may contribute to breaking the jasmine test.

Reply:
Let's go with `entry`

![image](https://user-images.githubusercontent.com/6675409/65352710-46840280-dbb9-11e9-8117-6e8d948fc414.png)

Reply:
Nicely done here :+1: